packages/buendia-dashboard/data/usr/share/buendia/tests/40-dashboard-running

```diff
@@ -0,0 +1,14 @@
+test_10_run_dashboard_cron () {
+    execute_cron_right_now dashboard
+    # FIXME: ^^ this throws an error when run manually:
+    # $ sudo /usr/sbin/logrotate /etc/logrotate.d/buendia-dashboard
+    # error: /etc/logrotate.d/buendia-dashboard:27 error verifying olddir path /var/log/requests/old: No such file or directory
+    true
+}
+
+test_20_dashboard_says_server_is_running () {
+    # When a user visits the dashboard page,
+    # Then they are told that the server is serving.
+    curl http://localhost:9999/ > dashboard.htm
+    grep SERVING dashboard.htm
```

Comment:
[taste] `curl | grep` in one pipeline so there aren't temporary files left over?

Reply:
The test cases are run in a temp directory, so this is a non-issue. I've documented that behavior over in test-harness.sh now tho